### DISCUSSION TITLE
More Information on how to implement this with Webflow

### DISCUSSION THREAD

**Discussion by Johnsonnnnnnn** [elbwalker/walkerOS#141] [General]
Hi, i am struggling to implement this to my webflow.

I have followed this - https://www.elbwalker.com/blog/tracking-on-webflow#prerequisites, added the script and followed along but when i run GTM tag assistant no dataLayer calls are being made with the attributes i have set?

I really hope someone can help me.

**Comment by aylaju** [comment]
Hi! Please try and add `data-default="true"` attribute to the script, it's missing in the one provided in the blog. You can also copy it from here: https://dev.to/walkerjs/webflow-walkerjs-1oml

Let me know if it works! I will edit the tutorial in our blog. 🏃‍♀️
